src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private static final String NAME = "circle";

```

Comment:
What is the reason to have this constant?
```suggestion
```

Reply:
if we add new methods, we will use the same variable - that's why I created this variable